lib/api/objectDelete.js

```diff
@@ -121,6 +121,28 @@ function objectDelete(authInfo, request, log, cb) {
                 newDeleteMarker: false,
             };
             if (delOptions && delOptions.deleteData) {
+                if (bucketMD.isNFS() &&
```

Comment:
So this can be combined in the main if conditional without having another level?

Reply:
There is a sub-condition for NFS, otherwise it treats the operation like a regular delete (as was being done before).

Reply:
It would be easy to bump it up an indentation level for readability 😄.